parsec/parsec.c

```diff
@@ -2759,3 +2759,50 @@ void parsec_set_my_execution_stream(parsec_execution_stream_t *es)
 {
     PARSEC_TLS_SET_SPECIFIC(parsec_tls_execution_stream, es);
 }
+
+
+/**
+ * Query the number of devices of the requested type available in the
+ * PaRSEC context.
+ */
+int parsec_context_query(parsec_context_t *context, parsec_context_query_cmd_t cmd, ...)
+{
+    parsec_device_module_t* dev;
+    va_list args;
+    va_start(args, cmd);
```

Comment:
Missing call to `va_end`

Reply:
There are too many return points to place an va_end. Plus everything related to the va_list is on the stack, so returning should clean up everything.